state/state.go

```diff
@@ -1056,15 +1058,15 @@ func (st *State) AddApplication(args AddApplicationArgs) (_ *Application, err er
 
 	// CAAS charms don't support volume/block storage yet.
 	if model.Type() == ModelTypeCAAS {
-		for name, stor := range args.Charm.Meta().Storage {
-			if storageKind(stor.Type) != storage.StorageKindBlock {
+		for name, store := range args.Charm.Meta().Storage {
```

Comment:
Calling it "stor" was deliberate as it is a contraction of "storage"
"store" implies something else
If you want to renamed, maybe "charmStorage" or "storage"